ecs-init/config/development.go

```diff
@@ -1,51 +0,0 @@
-//go:build development
```

Comment:
What's the purpose of this file? Why are we removing it as part of this pull request?

Reply:
This file could be left in, the main problem is the "dev" make target automatically added the 'development' tag, which was also removed in this commit so I guess that pretty much solves it.

I don't mind if we just leave it though those commits are from 8.5 years ago and I've not encountered anyone using this in my 5 years on the team...

Reply:
Good call out!! Updated summary with the details.